src/docs/DesignTokens.stories.mdx

```diff
@@ -0,0 +1,70 @@
+import { Meta } from '@storybook/addon-docs';
+import { DesignTokenDocBlock } from 'storybook-design-token/dist/doc-blocks';
+
+<Meta
+    title="Docs / Design Tokens"
+    parameters={{
+        viewMode: 'docs'
+    }}
+/>
+
+# Design Tokens
+
+Design tokens are name and value pairings that represent small, repeatable design decisions. Design tokens use meaningful names to represent visual styles, instead of hard-coded values.
+
+For example, the background color for a card uses the design token `card.color.background`, instead of a specific color value like `#FFFFFF`.
+
+Below you can see our design tokens grouped by type. Because these tables are generated from SCSS file, variable name format is different from the usage in code. In Styled components you should use it with dot notation and without $ sign.
+E.g. `$ref-color-purple-2` = `ref.color.purple.2`
+
+## Color
+
+<DesignTokenDocBlock showSearch={false} categoryName="Color" maxHeight={9999} />
+
+## FontWeight
+
+<DesignTokenDocBlock showSearch={false} categoryName="FontWeights" />
```

Comment:
Please add `maxHeight` prop to all categories. First of all, some of the categories already need it now and secondly, you'll never know how many tokens will each category contain in the future 😊

Reply:
Ok, added :)